besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -3630,10 +3630,17 @@ private String generateConfigurationOverview() {
           .setRpcHttpApis(jsonRpcConfiguration.getRpcApis());
     }
 
-    if (engineJsonRpcConfiguration != null && engineJsonRpcConfiguration.isEnabled()) {
-      builder
-          .setEnginePort(engineJsonRpcConfiguration.getPort())
-          .setEngineApis(engineJsonRpcConfiguration.getRpcApis());
+    if (engineJsonRpcConfiguration != null) {
+      if (engineJsonRpcConfiguration.isEnabled()) {
```

Comment:
Why are these broken up?

I'm struggling to imagine a scenario where jwt token is needed without the engine api being enabled.

Reply:
Think you're right. Refactored this. Thank you